app/konnect/getting-started/productize-service.md

```diff
@@ -0,0 +1,74 @@
+
+![{{site.konnect_short_name}} service diagram](/assets/images/docs/konnect/konnect-services-diagram.png)
+
+This guide will walk you through creating an API product and productizing it by deploying it to the Dev Portal.
+
+## Prerequisites
+
+If you're following the {{site.konnect_short_name}} quickstart guide,
+make sure you have [configured a runtime](/konnect/getting-started/configure-runtime/).
+
+## Create an API product <!--these steps will need to be verified once customer 0 happens-->
+
+Let's set up your first API product and API product version, by clicking {% konnect_icon api-product %} [**API Products**](https://cloud.konghq.com/apiproducts) from the {{site.konnect_short_name}} side navigation bar.
+
+1. Select **API Product** from the API products dashboard to add a new API product.
+
+1. Create a new name for your API product, and enter an optional **Description** and any **labels** that you want to associate with the product, then press **create**. 
+
+You will be greeted by the dashboard for the API product that you just created. You can use this dashboard to manage an API product. You can read more about this dashboard on our API products [overview page](/konnect/api-products/)
+
+### Create an API product version
+
+After creating a new API product, you can attach an API product version to it.
+
+1. From the API product builder, select **Product Versions**, then select **New Version**.
+
+1. Enter a version name. For example `v1`.
+     A version name can be any string containing letters, numbers, or characters;
+     for example, `1.0.0`, `v1`, or `version#1`. A service can have multiple
+     versions.
+1. Click **Create** to finish creating the product version and be taken to the **Product Versions dashboard**.
+
+After creating the new version, you will see **Link with a Gateway Service** as an option in the Product Version Dashboard. You can link a Gateway service to your product version to enable features like App registration. 
+
+1. Select **Link with a Gateway Service**. 
+
+    Choose a [runtime group](/konnect/runtime-manager/runtime-groups/) and [Gateway Service](/konnect/runtime-manager/configuration/#gateway-services) to
+    deploy this API product version to. This lets you deploy to a specific group of
+    runtime instances in a specific environment.
+
+    {:.note}
+    > **Note:** Application registration is only available for
+    services in the default runtime group, so if you plan on using
+    [application registration](/konnect/dev-portal/applications/application-overview/),
+    choose `default` in this step.
+
+    Different versions of the same API product can run in different runtime groups.
```

Comment:
@Guaris I suggest to remove this entire point + the bullet points underneath. There should be no real restrictions anymore since you can't link the same gateway service twice to different product versions.